modules/storage-ephemeral-storage-manage.adoc

```diff
@@ -40,17 +58,33 @@ spec:
     volumeMounts:
     - name: ephemeral
       mountPath: "/tmp"
   volumes:
     - name: ephemeral
       emptyDir: {}
 ----
-<1> Request for local ephemeral storage.
-<2> Limit for local ephemeral storage.
+<1> Container request for local ephemeral storage.
+<2> Container limit for local ephemeral storage.
+
+ifndef::microshift[]
+[id=storage-ephemeral-storage-scheduling-eviction_{context}]
+== Ephemeral storage configuration effects pod scheduling and eviction
+The settings in the pod spec affect both how the scheduler makes a decision about scheduling pods and when kubelet evicts pods.
+
+* First, the scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity of the node. In this case, the pod can be assigned to a node only if the node's available ephemeral storage (allocatable resource) is more than 4GiB.
+
+* Second, at the container level, because the first container sets a resource limit, kubelet eviction manager measures the disk usage of this container and evicts the pod if the storage usage of the container exceeds its limit (4GiB). The kubelet also marks the pod for eviction if the total usage exceeds the overall pod storage limit (8GiB).
+endif::microshift[]
 
-This setting in the pod spec affects how the scheduler makes a decision on scheduling pods, and also how kubelet evict pods. First of all, the scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity of the node. In this case, the pod can be assigned to a node only if its available ephemeral storage (allocatable resource) is more than 4GiB.
+ifdef::microshift[]
+[id=storage-ephemeral-storage-eviction_{context}]
+== Ephemeral storage configuration effects pod eviction
+The settings in the pod spec affect when kubelet evicts pods. At the container level, because the first container sets a resource limit, kubelet eviction manager measures the disk usage of this container and evicts the pod if the storage usage of the container exceeds its limit (4GiB). The kubelet also marks the pod for eviction if the total usage exceeds the overall pod storage limit (8GiB).
 
-Secondly, at the container level, since the first container sets resource limit, kubelet eviction manager measures the disk usage of this container and evicts the pod if the storage usage of this container exceeds its limit (4GiB). At the pod level, kubelet works out an overall pod storage limit by adding up the limits of all the containers in that pod. In this case, the total storage usage at the pod level is the sum of the disk usage from all containers plus the pod's `emptyDir` volumes. If this total usage exceeds the overall pod storage limit (4GiB), then the kubelet also marks the pod for eviction.
\ No newline at end of file
+This policy is strictly for `emptyDir` volumes and is not applied to persistent storage. You can specify the `priorityClass` of pods to exempt the pod from eviction.
```

Comment:
Does it makes sense to add this sentence after line 82? Line 81 is the same content as line 87, and both are wrapped for Microshift, so wondering if the repeated content is necessary, or if line 89 can be added to line 81.

Reply:
89 can't be added to 81 as that section is only going to the OpenShift docs. I do not know if the persistent storage caveat/option is valid there as well. I am only vetting for MicroShift with this PR.